### ISSUE TITLE
Cannot add custom connectors in 0.40.22

### ISSUE THREAD

**Issue by awoehrl** [airbytehq/airbyte#19802]
## Environment
- **Airbyte version**:  0.40.22
- **OS Version / Instance**: Debian 10
- **Deployment**: Docker
- **Source Connector and version**: Custom
- **Destination Connector and version**: 
- **Step where error happened**: Setup new Connector

## Current Behavior
I'm trying to add a custom connector, but it fails silently.
I open the "New connector" modal, fill out the fields and hit "Add". After that, I can see the loading icon. Then the modal closes without any info and I'm ending up at the settings/account page.
The source doesn't show up anywhere in the UI. However, I can see it via Octavia CLI.

I downgraded to 0.40.14 and there it is working fine.

## Expected Behavior
Custom connector should show up in UI and should be usable.

## Logs
```
airbyte-worker                      | 2022-11-24 10:33:25 INFO i.a.c.i.LineGobbler(voidCall):114 - Checking if redacted.pkg.dev/redacted//npg-airbyte-docker-connectors/plenigo-source:0.2 exists...
airbyte-worker                      | 2022-11-24 10:33:25 INFO i.a.c.i.LineGobbler(voidCall):114 - redacted.pkg.dev/redacted//npg-airbyte-docker-connectors/plenigo-source:0.2 was found locally.
airbyte-worker                      | 2022-11-24 10:33:25 INFO i.a.w.p.DockerProcessFactory(create):119 - Creating docker container = plenigo-source-spec-ce979a0a-0c83-4913-a759-f6fd056a415d-0-hpdba with resources io.airbyte.config.ResourceRequirements@1b780d57[cpuRequest=,cpuLimit=,memoryRequest=,memoryLimit=]
airbyte-worker                      | 2022-11-24 10:33:25 INFO i.a.w.p.DockerProcessFactory(create):163 - Preparing command: docker run --rm --init -i -w /data/ce979a0a-0c83-4913-a759-f6fd056a415d/0 --log-driver none --name plenigo-source-spec-ce979a0a-0c83-4913-a759-f6fd056a415d-0-hpdba --network host -v airbyte_workspace:/data -v /tmp/airbyte_local:/local -e DEPLOYMENT_MODE=OSS -e USE_STREAM_CAPABLE_STATE=true -e AIRBYTE_ROLE= -e WORKER_ENVIRONMENT=DOCKER -e WORKER_CONNECTOR_IMAGE=redacted.pkg.dev/redacted//npg-airbyte-docker-connectors/plenigo-source:0.2 -e WORKER_JOB_ATTEMPT=0 -e AIRBYTE_VERSION=0.40.22 -e WORKER_JOB_ID=ce979a0a-0c83-4913-a759-f6fd056a415d redacted.pkg.dev/redacted//npg-airbyte-docker-connectors
```
## Steps to Reproduce
1. Open Settings
2. Sources
3. Add new custom connector
4. Click Add

**Comment by octavia-squidington-iv** [comment]
cc @airbytehq/frontend

**Comment by josephkmh** [comment]
@awoehrl version 0.40.23 (released yesterday) contains a fix for this issue. Please give that a try!

**Comment by josephkmh** [comment]
The [releases](https://github.com/airbytehq/airbyte/releases) affected by this issue are: v0.40.19, v0.40.20, v0.40.21, v0.40.22. Closing this issue as it is resolved in v0.40.23.
